src/main/java/sungshin/sooon/service/PostService.java

```diff
@@ -0,0 +1,105 @@
+package sungshin.sooon.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import sungshin.sooon.dto.*;
+import sungshin.sooon.model.Post;
+import sungshin.sooon.model.Post_comment;
+import sungshin.sooon.model.Post_like;
+import sungshin.sooon.repository.PostCommentRepository;
+import sungshin.sooon.repository.PostLikeRepository;
+import sungshin.sooon.repository.PostRepository;
+
+@Service
+@RequiredArgsConstructor
+public class PostService {
+    @Autowired
+    private final PostRepository postRepository;
+
+    @Autowired
+    private final PostLikeRepository postLikeRepository;
+
+    @Autowired
+    private final PostCommentRepository postCommentRepository;
+
+    //게시글
+
+    @Transactional
+    public Long save(PostCreateRequestDto dto){
+        Post post = dto.toPost();
+        return postRepository.save(post).getPost_id();
+    }
+
+    @Transactional
+    public Long update(Long post_id, PostUpdateRequestDto dto){
+        Post post = postRepository.findById(post_id)
+                .orElseThrow(()->new IllegalArgumentException("[게시글:"+post_id+"] 해당 게시글이 없습니다."));
+        post.update(dto.getTitle(), dto.getContent(), dto.getIs_anonymous());
+        return post_id;
+    }
+
+    @Transactional
```

Comment:
```suggestion
    @Transactional(readOnly = true)
```
읽기 작업만 하는 경우에는 읽기전용으로 해야 성능이 향상된답니다
https://willseungh0.tistory.com/75